floss/main.py

```diff
@@ -543,6 +542,60 @@ def main(argv=None) -> int:
 
     language = identify_language(sample=sample, static_strings=static_strings)
 
+    if language == Language.GO:
+        enhancedStaticStrings = extract_go_strings(sample=sample, min_length=args.min_length)
+        enhancedStaticStrings = list(enhancedStaticStrings)
+
+        # Calculate String Coverage
+
+        all_static_string_list = []
+
+        for string_obj in static_strings:
+            if string_obj.string.isprintable():
+                all_static_string_list.append(string_obj.string)
+
+        ct = 0
+                
+        for string in all_static_string_list:
+            found = False
+            for j in enhancedStaticStrings:
+                k = j.string
+                if k in string:
+                    found = True
+                    break
+            if (not found) and (len(string) >= args.min_length):
+                ct += 1
```

Comment:
I think this algorithm has some issues:
- strings may be found multiple times, i.e., we need to remove found sequences
- I think the extracted strings should be in the outer loop
- let's also count the number of characters (by length of strings) covered

Reply:
there's also duplication/conflict now to https://github.com/mandiant/flare-floss/pull/810, let's incorporate https://github.com/mandiant/flare-floss/pull/810 into this PR or do you see value in having the extra analysis script?